Lib/test/test_dis.py

```diff
@@ -1050,12 +1105,12 @@ def test_explicit_first_line(self):
         self.assertEqual(list(actual), expected_opinfo_outer)
 
     def test_source_line_in_disassembly(self):
-        # Use the line in the source code
-        actual = dis.Bytecode(simple).dis()[:3]
+        # Use the line in the source code (split extracts the line no)
+        actual = dis.Bytecode(simple).dis().split(" ")[0]
```

Comment:
[:3] raises now (since this is the patch that puts the line over 999) and [:4] would raise if it's ever driven above 9999; that would be a complex test suite (!) but I'll go with .partition (good call on that vs. .split; thanks!)